### DISCUSSION TITLE
Google play is not working

### DISCUSSION THREAD

**Discussion by zakaria-ist** [A-Tokyo/react-native-stripe-checkout-webview#36] [General]
In my APP I cannot see the google pay & apple pay button. I tried in both android & iOS devices.

I commented this line also, but the issue does not solve.
// [wkWebViewConfig.userContentController addUserScript:script]; // this line that inject "html5HistoryAPIShimSource" on start

Does anyone have any idea?

**Comment by a-tokyo** [comment]
Thanks for starting the discussion @zakaria-ist ✨ 

Related issue https://github.com/A-Tokyo/react-native-stripe-checkout-webview/issues/4

**Comment by a-tokyo** [comment]
Update: this is a react-native-webview issue, we will try to solve it there!
